libcontainer/cgroups/fs/freezer.go

```diff
@@ -65,6 +65,10 @@ func (s *FreezerGroup) Set(path string, r *configs.Resources) (Err error) {
 				return err
 			}
 
+			if i%25 == 24 {
+				// A short sleep before reading back also helps.
```

Comment:
Could you update the comment to clarify this helps what

Reply:
There's a big comment above telling the whole story...

Ah, I just found it now contradicts what I say here. Will fix.

Reply:
Comments rewritten; PTAL @AkihiroSuda